sass/base/_css_variables.scss

```diff
@@ -6,8 +6,8 @@
     --button-color: #fff;
     --center-channel-bg: #fff;
     --center-channel-color: #3d3c40;
-    --dnd-indicator: #f74343;
-    --error-text: #fd5960;
+    --dnd-indicator: #d24b4e;
+    --error-text: #d24b4e;
```

Comment:
do these values align with the design system?

Reply:
yes! those were already added before.